app/src/main/res/values/strings.xml

```diff
@@ -5,8 +5,8 @@
     <string name="action_search">Search</string>
     <string name="action_add">Add</string>
 
-    <string name="noGiftCards">You don\'t have any cards at the moment. Click the "+" (plus) button to get started.\n\nCatima lets you carry your cards on your phone, so they are always within reach.</string>
-    <string name="noMatchingGiftCards">No cards match the filter. Please try some different terms.</string>
+    <string name="noGiftCards">Click the "+" (plus) button to add a card first.\n\nCatima carries your cards on the device, so they are always within reach.</string>
+    <string name="noMatchingGiftCards">Didn't find anything. Please try again, or look elsewhere.</string>
```

Comment:
Short of one string for each, there has got to be a limit to how many exact retries "try again" warrants. (?)

Reply:
Well, 0. Trying again won't change anything. I'll just add an extra string later, just make this only be about searching.